compiler/src/dotty/tools/MainGenericRunner.scala

```diff
@@ -241,4 +243,22 @@ object MainGenericRunner {
     e.foreach(_.printStackTrace())
     !isFailure
   }
+
+  def display(settings: Settings)= Seq(
```

Comment:
Isn't it dead code? I guess it's for debugging purposes, maybe we could add some flag for displaying this debug, or let's remove it

Reply:
How about an environment variable, maybe something like this:
```scala
   lazy val debug = Option(System.getenv("RUNNER_DEBUG")) != None
   [...]
   if debug then printf("settings: %s\n", display(settings))
```
I also don't mind removing it, I can always add it back temporarily, if needed.

Reply:
I think we can remove it.